packages/api-server/src/cliHandlers.ts

```diff
@@ -43,60 +43,67 @@ export const apiCliOptions = {
 export const apiServerHandler = async (options: ApiServerArgs) => {
   const { port, socket, apiRootPath } = options
   const tsApiServer = Date.now()
-  process.stdout.write(c.dim(c.italic('Starting API Server...\n')))
+  process.stdout.write(c.dim.italic('Starting API Server...\n'))
 
   let fastify = createFastifyInstance()
 
   // Import Server Functions.
   fastify = await withFunctions(fastify, options)
 
-  const http = startFastifyServer({
-    port,
-    socket,
-    fastify,
-  }).ready(() => {
-    console.log(c.italic(c.dim('Took ' + (Date.now() - tsApiServer) + ' ms')))
-
-    const on = socket
-      ? socket
-      : c.magenta(`http://localhost:${port}${apiRootPath}`)
-    console.log(`API listening on ${on}`)
-    const graphqlEnd = c.magenta(`${apiRootPath}graphql`)
-    console.log(`GraphQL endpoint at ${graphqlEnd}`)
+  fastify = await startFastifyServer({ port, socket, fastify })
+
+  fastify.ready(() => {
+    console.log(c.dim.italic('Took ' + (Date.now() - tsApiServer) + ' ms'))
+
+    // In the past, in development, we've prioritized showing a friendlier
+    // host than the listen-on-all-ipv6-addresses '[::]'. Here we replace it
+    // with 'localhost' only if 1) we're not in production and 2) it's there.
+    // In production it's important to be transparent.
+    //
+    // We have this logic for `apiServerHandler` because this it' the only
+    // handler called by the watch bin (which is called by `yarn rw dev`).
+    let address = fastify.listeningOrigin
+    if (process.env.NODE_ENV !== 'production') {
+      address = address.replace(/http:\/\/\[::\]/, 'http://localhost')
+    }
```

Comment:
@Tobbe mentioned we prioritized showing a friendly host in dev so I've added this logic here and to server.ts so that in dev, you see:

```
api | 16:36:21 🌲 Server listening at http://localhost:8911
api | Took 229 ms
api | API server listening at http://localhost:8911/
api | GraphQL endpoint at http://localhost:8911/graphql
```